qutebrowser/components/caretcommands.py

```diff
@@ -209,3 +209,10 @@ def follow_selected(tab_obj: apitypes.Tab, *, tab: bool = False) -> None:
         tab_obj.caret.follow_selected(tab=tab)
     except apitypes.WebTabError as e:
         raise cmdutils.CommandError(str(e))
+
+
+@cmdutils.register()
+@cmdutils.argument('tab', value=cmdutils.Value.cur_tab)
+def reverse_selection(tab: apitypes.Tab) -> None:
+    """Toggle caret selection mode."""
```

Comment:
That seems wrong.

Reply:
Oh, yes. Sorry, I copied and pasted from elsewhere and forgot to change the docstring. Thanks.